.gitignore

```diff
@@ -1,2 +1,5 @@
 build
-node_modules
\ No newline at end of file
+node_modules
+**/*.js
```

Comment:
To be more specific let's ignore `lib` and then add them back in to package.json so both the TS and JS files are published to the module https://github.com/Tyriar/node-pty/blob/ad71e8989d7613e2e439cc191ede6d5e48600f58/package.json#L15-L21